app/src/sharedTest/java/org/oppia/android/app/testing/NavigationDrawerActivityTest.kt

```diff
@@ -278,8 +284,400 @@ class NavigationDrawerActivityTest {
     }
   }
 
+  // TODO(#2535): Unable to open NavigationDrawer multiple times on Robolectric
+  @RunOn(TestPlatform.ESPRESSO)
+  @Test
+  fun testNavDrawer_openNavDrawer_switchProfile_cancel_homeIsSelected() {
```

Comment:
The current flow is in the latest develop is:
1. Open nav drawer
2. click options
3. Open nav drawer
4. click switch profile
5. cancel - nav drawer gets closed and we are on the same options screen but,
6. open nav drawer
7. home gets selected (issue)

Reply:
Yes, we have test cases for this below.
This test case ensures that the right behavior is exhibited for home.